microsetta_interface/templates/survey.jinja2

```diff
@@ -5,11 +5,60 @@
     <script type="text/javascript" src="/static/vendor/js/vue-2.5.17.min.js"></script>
     <script type="text/javascript" src="/static/vendor/vue-form-generator-2.3.4/vfg.js"></script>
     <link rel="stylesheet" type="text/css" href="/static/vendor/vue-form-generator-2.3.4/vfg.css">
+    <style type="text/css">
+        .vue-form-generator .field-radios .radio-list label {
```

Comment:
My understanding is that this catches labels in groups of radio buttons, similar style be set for groups of checkboxes?  

(See Do you eat a specialized diet ? (select all that apply), Step 2 of 7 in the Microsetta Participant Survey)

Rough guess on the css selector is:
.vue-form-generator .field-checklist .dropList .list-row label

Could be a separate change entirely though.

Reply:
Are you thinking you'd want to have the checkbox options span the entire width of the screen (similar to radio buttons), rather than in a one-per-line dropdown?

Reply:
@dhakim87 is this a blocking item or can this be sorted out later?